components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3600,6 +3600,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         return [];
     }
 
+    async reportErrorBoundary(ctx: TraceContextWithSpan, url: string, message: string): Promise<void> {
+        log.info("dashboard error boundary", { message, url, userId: this.user?.id });
```

Comment:
I'd do Warn here. We can verify if it's a legit error, or abuse of the endpoint but assuming most invocations are legitimate, warn seems appropriate.